frontend/src/Components/NewRunConfig/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Checkbox, FormGroup, NumericInput } from "@blueprintjs/core";
+import { SubmitSimulation } from "../NewRunState";
+import { advancedParams, params } from "../ParameterLabels";
+import { SimConfig } from "../../../Helpers/SimConfig";
+import "./AdvancedSettings.css";
+
+export default function AdvancedSettingsMenu(state: any) {
```

Comment:
You can pass an interface with the props definition to ensure the type of state[0] and state[1] is always the same and not dependent on ordering